packages/trader/src/App/Components/Elements/ContractAudit/contract-details.jsx

```diff
@@ -117,16 +122,35 @@ const ContractDetails = ({ contract_end_time, contract_info, duration, duration_
                                 value={getBarrierValue(contract_info) || ' - '}
                             />
                         )}
+                        {hasTwoBarriers(contract_type) && (
+                            <React.Fragment>
+                                {[high_barrier, low_barrier].map((barrier, index) => (
+                                    <ContractAuditItem
+                                        id={`dt_bt_label_${index + 1}`}
+                                        icon={<Icon icon='IcContractStrike' size={24} />}
+                                        key={barrier}
+                                        label={
+                                            high_barrier === barrier
+                                                ? localize('High barrier')
+                                                : localize('Low barrier')
+                                        }
+                                        value={barrier}
+                                    />
+                                ))}
+                            </React.Fragment>
+                        )}
                         {show_payout_per_point && (
                             <ContractAuditItem
                                 id='dt_bt_label'
                                 icon={<Icon icon='IcContractPayout' size={24} />}
                                 label={localize('Payout per point')}
                                 value={`${display_number_of_contracts} ${getCurrencyDisplayCode(currency)}` || ' - '}
```

Comment:
@kate-deriv `should_format={!is_vanilla}` is not needed here. Plus, there is no such prop in ContractAuditItem :)